src/components/Choropleth.vue

```diff
@@ -1,17 +1,17 @@
 <template>
     <div>
-        <LMap ref="map" style="height: 100vh; width: 100%" @ready="updateBounds">
+        <LMap ref="map" style="height: 100vh; width: 100%" @update:bounds="boundsUpdated">
             <LTileLayer data-testid="tile-layer" v-bind="backgroundLayer"></LTileLayer>
             <LGeoJson
-                v-for="f in featuresWithColours"
+                v-for="f in featuresWithColoursArr"
                 ref="featureRefs"
                 :key="getFeatureId(f.feature)"
                 :data-testid="getFeatureId(f.feature)"
```

Comment:
No great value in memo-izing this?

Reply:
I guess I must have used that in a test, but yeah it's a lot to render for the sake of a test if you've got thousands of the things..

Reply:
I tried memo-izing to see if it made a difference to performance for Brazil, but I didn't notice one (though I didn't actually measure the time)

Reply:
Sorry, I misunderstood your comment - yeah, I doubt it would make a difference since the array itself is changing. It might do if we kept the v-for for the admin1 regions, and just updated the visibility of the selected country ones... I think what Mantra's attempting will probably be more effective (if a bit less readable)...